src/controllers/user.controller.js

```diff
@@ -1 +1,32 @@
-exports.status = async (req, res, next) => res.json({ message: 'User endpoint is working.' })
\ No newline at end of file
+const User = require('../models/user.model')
```

Comment:
don't forget your semicolons...

Reply:
add it as required in eslint to avoid this issue next time

Reply:
eslint isn't configured for semilcolons yet, I will add the configuration and fix all semicolons in another pr. If I did that here this PR would be very messy.